server/settings/loaders.py

```diff
@@ -110,6 +110,9 @@ class ReferenceLoaderModel(BaseSettingsModel):
     display_handle: bool = SettingsField(
         title="Display Handle On Load References"
     )
+    lock_animation_instance_on_load: bool = SettingsField(
+        title="Lock Animation Instance on Rig Load"
+    )
```

Comment:
@LiborBatek how do you feel about this naming `lock_animation_instance_on_load`? Is that clear enough?

It's actually the _rig_ being loaded and the animation instance being created - so it feels a bit mixed.